lib/rbs_rails/active_record.rb

```diff
@@ -60,6 +60,54 @@ def generate
         sql_type_to_class(col.type)
       end
 
+      private def generated_finder_class_methods_decl
+        <<~RBS
+          module GeneratedFinderClassMethods
```

Comment:
Note: This code is based on activerecord-6.0.3.2 in gem_rbs_collection
https://github.com/ruby/gem_rbs_collection/blob/551969504a000b3a2f82dc32e4cb3dbc40ea2285/gems/activerecord/6.0.3.2/activerecord.rbs#L102